setup.py

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env python
+
+from distutils.core import setup
+
+setup(
+    name='condor-comforter',
+    version='0.1',
+    description="Some helper scripts for running HTCondor jobs",
+    author='Robin Aggleton',
+    author_email='',
+    url='https://github.com/BristolComputing/condor-comforter',
+    py_modules=['cmsRunCondor.cmsRunCondor', 'haddaway.haddaway'],
+    scripts=['cmsRunCondor/cmsRunCondor.py', 'haddaway/haddaway.py'],
+    install_requires=['htcondenser>=0.3.0'],
+    dependency_links=['git+https://github.com/raggleton/htcondenser#egg=htcondenser-0.3.0']
```

Comment:
If you try without the `-0.3.0`, you get

```
  Could not find a version that satisfies the requirement htcondenser>=0.3.0 (from condor-comforter==0.2) (from versions: )
No matching distribution found for htcondenser>=0.3.0 (from condor-comforter==0.2)
```
so it looks like it requires telling which version to use.

See also: https://setuptools.readthedocs.io/en/latest/setuptools.html#dependencies-that-aren-t-in-pypi 

> In the case of a VCS checkout, you should also append #egg=project-version in order to identify for what package that checkout should be used.